### PR TITLE
feat: updated examples for swagger

### PR DESCRIPTION
**Pull request by aanshi07**
## What are the changes introduced in this PR?

Updated examples for swagger for router and processor.

Screenshots for Processor ,Router and Batch:

Processor:
Screenshot 2024-07-18 at 4 45 42 PM

Router:
Screenshot 2024-07-18 at 4 46 20 PM

Batch:
Screenshot 2024-07-18 at 4 46 30 PM

## What is the related Linear task?

Resolves INT-2274

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

@coderabbitai review

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added in **new readability format**?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.
Screenshot 2024-07-18 at 4 46 30 PM

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3526/test-report.html

File: swagger/components/examples/processor/failure.yaml
Comment (krishna2020) [inline swagger/components/examples/processor/failure.yaml:1]: can we re-check if all the keys mentioned in `metadata` of example value are being sent by rudder-server ?

image

File: swagger/components/examples/router/success.yaml
Comment (krishna2020) [inline swagger/components/examples/router/success.yaml:1]: Check the example input for routerTransform, `request: {}` is not valid

image

Comment (Gauravudia) [reply swagger/components/examples/router/success.yaml:1]: request is an optional parameter in Router transform input
https://github.com/rudderlabs/rudder-transformer/blob/2eb92e3332ef0e8b2f83621fe0130fbc1356fa91/src/types/index.ts#L159

File: swagger/components/examples/processor/failure.yaml
Comment (aanshi07) [reply swagger/components/examples/processor/failure.yaml:1]: yes @krishna2020 I am checking on this !

File: swagger/components/examples/router/success.yaml
Comment (krishna2020) [reply swagger/components/examples/router/success.yaml:1]: Then, we can update it with details
`request: { "query" : <ctx.request.query> }`
Ref: https://github.com/rudderlabs/rudder-transformer/blob/72d696541d3657b6e69c5b794895e3107bdb56fe/src/services/destination/preTransformation.ts#L13

Comment (Gauravudia) [reply swagger/components/examples/router/success.yaml:1]: @aanshi07 You can create request.yaml with query in schema, and use it in routerTransformationRequest.yaml

Comment (aanshi07) [reply swagger/components/examples/router/success.yaml:1]: okay @Gauravudia will do it.

**Comment by Gauravudia** [comment]
Screenshot 2024-07-18 at 10 27 48 AM
@aanshi07 schema/request is not rendering for router transform. Check the swagger code once.

**Review by Gauravudia — APPROVED** [review]

**Review by krishna2020 — APPROVED** [review]

**Pull request merged**